query/src/main/java/org/apache/kylin/query/adhoc/PushDownRunnerJdbcImpl.java

```diff
@@ -214,6 +214,9 @@ private SelectedColumnMeta extractColumnMeta(ResultSetMetaData resultSetMetaData
             // Fall back to default value
         }
 
+        columnLabel = columnLabel.toUpperCase(Locale.ROOT);
```

Comment:
Why do upper case convert here?
It seems that there is possibility "columnLabel" or "columnName" be null if SQLException caught.